src/flb_upstream.c

```diff
@@ -237,6 +237,39 @@ struct flb_upstream *flb_upstream_create(struct flb_config *config,
     return u;
 }
 
+/*
+ * Checks whehter a destinate URL should be proxied.
+ */
+int flb_should_proxy_for_host(const char *host, const char *proxy, const char *no_proxy)
```

Comment:
the function name does not follow the interface pattern, please rename it to something like ```flb_upstream_needs_proxy()```

Reply:
DONE.